shared/macros/10-bash.jinja

```diff
@@ -1722,7 +1722,7 @@ Ensures that given mount point is in :code:`/etc/fstab`.
 
 #}}
 {{% macro bash_ensure_mount_option_in_fstab(mount_point, mount_opt, fs_spec, type) -%}}
-mount_point_match_regexp="$(printf "[^#].*[[:space:]]%s[[:space:]]" {{{ mount_point }}})"
+mount_point_match_regexp="$(printf "^[[:space:]]*[^#].*[[:space:]]%s[[:space:]]" {{{ mount_point }}})"
```

Comment:
I would reduce this to:
`^[[:space:]]*[^#].*%s[[:space:]]`

Reply:
I would keep it as it is... we are ensuring that we do not grep for something like /mnt/image/boot instead of /boot.